### PR TITLE
Update ko.js

### PR DESCRIPTION
**Pull request by starniah-web3**
1. when it is more natural to write in English than in Korean, write in English.
 - 'Doge' is translated in various ways, unified as Doge.
- Words in social, wallet transfer function, etc.
- Unified 'Dogverse' in English.

2. I tried to be consistent with the words used by Korean exchanges and existing wallet apps.
3. For the social part, I unified the X and words as much as possible.
4. If I need to make any changes while using the app in the future, I will make additional changes.

### CODE REVIEW

**Comment by alexanvl** [comment]
> 1. when it is more natural to write in English than in Korean, write in English.
> 
> * 'Doge' is translated in various ways, unified as Doge.
> * Words in social, wallet transfer function, etc.
> * Unified 'Dogverse' in English.
> 
> 2. I tried to be consistent with the words used by Korean exchanges and existing wallet apps.
> 3. For the social part, I unified the X and words as much as possible.
> 4. If I need to make any changes while using the app in the future, I will make additional changes.

Looks great, thank you!

**Review by alexanvl — APPROVED** [review]

File: ko.js [hunk 1]
```diff
@@ -54,66 +54,66 @@ export const ko = {
...
     tx_hash: `TX 해시`,
     tx_hash_copied: `거래 해시가 복사되었습니다.`,
     note_copied: `메모가 복사되었습니다.`,
     address_copied: `주소가 복사되었습니다`,
     wallet: `지갑`,
-    dogeverse: `도지버스`,
-    shop: `가게`,
-    receive_doge: `총독을 받다`,
-    send_doge: `총독 보내기`,
+    dogeverse: `dogeverse`,
```
Comment (alexanvl) [inline ko.js]: Please capitalize

File: ko.js [hunk 2]
```diff
@@ -54,66 +54,66 @@ export const ko = {
...
+    shop: `shop`,
```
Comment (alexanvl) [inline ko.js]: Please capitalize

File: ko.js [hunk 3]
```diff
@@ -54,66 +54,66 @@ export const ko = {
...
-    add_new: `새로운 걸 더하다`,
+    add_new: `새 주소 추가`, //Understood as a menu to add an address to the user's address book.
     sending_to: `다음으로 보내는 중`,
-    save: `구하다`,
+    save: `저장하기`,
     saved: `저장됨`,
-    balance: `균형`,
-    max: `맥스`,
+    balance: `잔고`,
+    max: `max`,
```
Comment (alexanvl) [inline ko.js]: Please capitalize

File: ko.js [hunk 4]
```diff
@@ -195,81 +195,81 @@ export const ko = {
...
+    some_doge: `작은 Doge 팁!`,
     total: `총`,
     done: `완료`,
     shop_with_doge: `Doge와 함께 쇼핑하세요`,
     oops: `이런!`,
     no_comments_yet: `아직 댓글이 없습니다`,
     to_post_a_comment: `댓글을 게시하려면`,
     your_comment: `귀하의 의견`,
-    post: `우편`,
+    post: `post`,
```
Comment (alexanvl) [inline ko.js]: Please capitalize

**Review by alexanvl — CHANGES_REQUESTED** [review]
Thank you for your submission. Please make the requested capitalization changes and I will approve and merge it.

**Comment by starniah-web3** [comment]
Done. This was my first time collaborating on github and it was a great experience. Thx.

**Review by alexanvl — APPROVED** [review]

**Pull request merged**